src/request.rs

```diff
@@ -110,6 +116,7 @@ pub(crate) async fn request<
 
     let headers = Headers::new().unwrap();
     headers.append("Authorization: Bearer", apikey).unwrap();
+    headers.append("User-Agent", user_agent).unwrap();
```

Comment:
This is also going to error on the wasm check because `user_agent` is not defined in this scope. In addition `Headers` wants a `&str` instead of a `String` for the user agent.

```rust
let user_agent = qualified_version();

let headers = Headers::new().unwrap();
headers.append("Authorization: Bearer", apikey).unwrap();
headers.append("User-Agent", &user_agent).unwrap();
```